### DISCUSSION TITLE
Different logos for login and toolbar

### DISCUSSION THREAD

**Discussion by HenriqueSPin** [laravel/nova-issues#4515] [Ideas & Feature Requests]
Hi there! It would be nice to have different configuration options for the login logo and the toolbar logo.

So, we can use different versions (short or full logo version) and dimensions (bigger in login) of our brand logo.

```php
    'brand' => [
        'logo' => resource_path('/img/logo.svg'),
        'login_logo' => resource_path('/img/logo_full.svg'),
    ],
```

If no ```nova.brand.login_logo``` is set, it will fallback for the ```nova.brand.logo```.

What do you guys think of this?

**Comment by geanruca** [comment]
Do we have this? 🙏🏽
